src/Workspaces/Core/Portable/Workspace/Solution/DocumentState_LinkedFileReuse.cs

```diff
@@ -85,12 +86,13 @@ public DocumentState UpdateTextAndTreeContents(ITextAndVersionSource siblingText
                 ParseOptions parseOptions,
                 SyntaxNode siblingRoot,
                 VersionStamp siblingVersion,
+                bool force,
                 [NotNullWhen(true)] out TreeAndVersion? newTreeAndVersion)
             {
                 var siblingTree = siblingRoot.SyntaxTree;
 
                 // Look for things that disqualify us from being able to use our sibling's root.
-                if (!CanReuseSiblingRoot())
+                if (!CanReuseSiblingRoot(force))
```

Comment:
so i did think about that.  but i personally found it clearer.  esp since it meant any callers of CanReuseSiblingRoot would ahve to update.  i'm ok with eitehr.  but this was intentional.